### PR TITLE
Translation of cms chapter to Japanese

### PR DESCRIPTION
**Pull request by ksakae1216**
Part of #586 

Please review.

### CODE REVIEW

File: src/content/ja/2019/cms.md [hunk 1]
```diff
@@ -0,0 +1,663 @@
+---
+part_number: III
+chapter_number: 14
+title: CMS
+description: 2019年版Web年鑑CMS章では、CMSの採用、CMS組み合わせの構築方法、CMSを搭載したWebサイトのユーザーエクスペリエンス、CMSのイノベーションを取り上げています。
+authors: [ernee, amedina]
+reviewers: [sirjonathan]
+translators: [ksakae]
+discuss: 1769
```
Comment (bazzadp) [inline src/content/ja/2019/cms.md:9]: We added extra meta data in #728 

```suggestion
discuss: 1769
results: https://docs.google.com/spreadsheets/d/1FDYe6QdoY3UtXodE2estTdwMsTG-hHNrOe9wEYLlwAw/
queries: 14_CMS
```

Comment (ksakae1216) [reply src/content/ja/2019/cms.md:9]: Thanks, I fixed.

**Comment by bazzadp** [comment]
@ksakae1216 as we haven't merged this should we do the extra changes for this chapter (bio and featured_chapter changes) to this pull request?

Then can do an separate pull request for all the other missing ones for chapters that have been translated already.

**Comment by bazzadp** [comment]
About to do a release so will merge this in. You can do the extra changes in that additional PR you were talking about.

**Pull request merged**